src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -253,48 +259,27 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
     parameters: Omit<RapSwapActionParameters<typeof type>, 'gasParams' | 'gasFeeParamsBySpeed' | 'selectedGasFee'>;
   }) => {
     try {
-      const NotificationManager = IS_IOS ? NativeModules.NotificationManager : null;
-      NotificationManager?.postNotification('rapInProgress');
+      const degenMode = swapsStore.getState().degenMode;
+      const atomicSwapsEnabled =
+        (atomicSwapsExperimentalFlag || config.atomic_swaps_enabled) &&
+        (await getShouldDelegate(parameters.chainId, parameters.quote as Quote | CrosschainQuote, parameters.assetToSell));
+      const shouldDelegate = atomicSwapsEnabled && (await getShouldDelegate(parameters.chainId, parameters.quote, parameters.assetToSell));
```

Comment:
```suggestion
      const atomicSwapsEnabled =
        atomicSwapsExperimentalFlag || config.atomic_swaps_enabled;
      const shouldDelegate = atomicSwapsEnabled && (await getShouldDelegate(parameters.chainId, parameters.quote, parameters.assetToSell));
```

this should be equal, no? and faster